### ISSUE TITLE
1.3.4, Does this pass?

### ISSUE THREAD

**Issue by anevins12** [w3c/wcag#3228]
I created an issue about 1.3.4 and the piano exception recently, but had a skewed understanding and I led a couple of folks down rabbit holes. Hopefully, this question will be better put across.

I want to know whether "essential" exceptions are allowed to ignore the orientation preference of the user, for example by always showing a landscape orientation, or, whether exceptions can be restricted or absent from one orientation.

If I provide the following scenarios, can you answer whether they pass 1.3.4?

1. An application for learning practical piano fingering shows the piano UI on landscape and portrait orientations, but the piano never changes orientation. For example, on landscape the longest side of the piano is on the longest side of the device, but on portrait the longest side of the piano is on the shortest side of the device. The piano UI does not rotate.
2. An application for learning practical piano fingering shows the piano UI on landscape, but on portrait hides the piano and instead shows a message "Rotate the device to view the piano."

**Comment by alastc** [comment]
Hi Andrew,

In your scenario 1, I'm a bit confused by "the piano never changes orientation", I think you mean it doesn't change relative to up/down (gravity), but does change relative to the device? I.e. the longest side of the keys changes depending on whether it is in portrait or landscape.

Assuming that, it passes the criteria as it has not restricted to the display orientation. However, in portrait it is probably un-playable due to the size of the keys. If they were the right size in landscape, they will be about 50% thinner in portrait.

In scenario 2 it is failing the SC unless you consider that functionality essential:
"if removed, would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way that would conform".

Can you achieve the functionality in another way? 

- Shrinking the keys would likely make it un-playable due to size.
- Having scrolling would not work if you have to play chords, or keys from either end within a certain time.

Therefore I think this (hypothetical) piano app would qualify for the essential exception.
